AST-349 - fix: Console error in case of Dropdown Toggle for Sticky Header on Mobile. (#2611)
brainstormforce/astra: changelog.txt

- Fix: Context Switch does not work if there are no elements in Primary Header for both Desktop & Mobile.
- Fix: Deprecated the Cart Style 'None' & merged it with existing option 'Outline' for WooCommerce and EDD.
- Fix: Accessibility - Tab navigation does not highlight menu with submenus.

> - Fix: Console error in case of Dropdown Toggle for Sticky Header on Mobile.

Reviewer: ```suggestion
- Fix: JavaScript error when clicking on the Dropdown Toggle for Sticky Header on Mobile.
```

Reviewer: - It's a `Javascript` error which is shown in the console'
- Avoid using words like `case`, `issue` etc and instead describe what the actual case or the issue is.

Author: Okay! Updated & Noted.
